docs/source/development/traffic_router.rst

```diff
@@ -143,6 +143,16 @@ To install the Traffic Router Developer environment:
 
 Once running, the :ref:`tr-api` is available over HTTP at http://localhost:3333 and over HTTPS at https://localhost:3443,  the HTTP routing interface is available on http://localhost:8888 and HTTPS is available on http://localhost:8443. The DNS server and routing interface is available on localhost:53 via TCP and UDP.
 
+Development Environment Upgrade
+===============================
```

Comment:
This should probably be a subsection of the Developer Environment section (using `------`) instead of a top-level section of the page (using `=====`).

Reply:
Done @ocket8888 , you can check it now.